### PR TITLE
feat(widget): customize images

### LINKED ISSUE #3979: [widget] Customise images
Allow to replace or remove some of the images. 

The params are already define. We need to implement and document.

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Fixes #3979 

1. Select "Limit" in Current trade type control

image

2. Scroll to Customization section and past an image URL into "Empty order image URL". For example: https://cow.fi/images/cowamm-illustration-lvr.svg

3. Scroll to "Your orders" in the widget preview

image

### CODE REVIEW

File: libs/widget-lib/src/types.ts [hunk 1]
```diff
@@ -204,11 +204,6 @@ export interface CowSwapWidgetParams {
    */
   theme?: CowSwapTheme | CowSwapWidgetPalette
 
-  /**
-   * Allows to set a custom logo for the widget.
-   */
-  logoUrl?: string
```
Comment (shoom3301) [inline libs/widget-lib/src/types.ts:210]: We never display logo in the widget

Comment (anxolin) [reply libs/widget-lib/src/types.ts:210]: I guess we will need to update the docs too once we are done with the changes https://docs.cow.fi/cow-protocol/tutorials/widget

**Review by anxolin — APPROVED** [review]
Works great! 

I don't think we need to add every param in the configurator. Or if we do, we should in the future hide a bunch of them, cause right now feels a bit overcomplicated 

image

**Comment by shoom3301** [comment]
@elena-zh thank you!

1. Will describe in the docs
2. Will describe in the docs
3. This is not necessary, this is responsibility of integrators

**Review by elena-zh — APPROVED** [review]
Approved then :)

**Pull request merged**